app/services/foreman/renderer/scope/macros/kickstart.rb

```diff
@@ -0,0 +1,118 @@
+          end
+          def kickstart_network(iface, host:, rhel_compatible:, os_major:, use_slaac:, static:, static6:)
+            return nil unless iface.is_a?(Nic::Managed)
+
+            network_options = []
+            nameservers = []
+            subnet4 = iface.subnet
+            subnet6 = iface.subnet6
+
+            # device and hostname
+            if iface.bond? && rhel_compatible && os_major >= 6
+              network_options.push("--device=#{iface.identifier}")
+            else
+              network_options.push("--device=#{iface.mac || iface.identifier}")
+            end
+            network_options.push("--hostname #{@host.name}")
+
+            # single stack
+            if subnet4 && !subnet6
+              network_options.push("--noipv6")
+            elsif !subnet4 && subnet6
+              network_options.push("--noipv4")
+            end
+
+            # dual stack MTU check
+            raise("IPv4 and IPv6 subnets have different MTU") if subnet4 && subnet6 && subnet4.mtu.present? && subnet6.mtu.present? && subnet4.mtu != subnet6.mtu
+
+            # mtu method is taking into account both ipv4 and ipv6 stacks
+            network_options.push("--mtu=#{iface.mtu}") if iface.mtu
+
+            # IPv4
+            if (subnet4 && !subnet4.dhcp_boot_mode?) || static
+              network_options.push("--bootproto static")
+              network_options.push("--ip=#{iface.ip}")
+              network_options.push("--netmask=#{subnet4.mask}")
+              network_options.push("--gateway=#{subnet4.gateway}")
+            elsif subnet4&.dhcp_boot_mode?
+              network_options.push("--bootproto dhcp")
+            end
+            if subnet4
+              nameservers.concat(subnet4.dns_servers)
+            end
+
+            # IPv6
+            if rhel_compatible && os_major >= 6
+              if (subnet6 && !subnet6.dhcp_boot_mode?) || static6
+                network_options.push("--ipv6=#{iface.ip6}/#{subnet6.cidr}")
+                network_options.push("--ipv6gateway=#{subnet6.gateway}")
+              elsif subnet6&.dhcp_boot_mode?
+                if use_slaac
+                  network_options.push("--ipv6 auto")
+                else
+                  network_options.push("--ipv6 dhcp")
+                end
+              end
+              if subnet6
+                nameservers.concat(subnet6.dns_servers)
+              end
+            end
+
+            # bond
+            if iface.bond? && rhel_compatible && os_major >= 6
+              bond_slaves = iface.attached_devices_identifiers.join(',')
+              network_options.push("--bondslaves=#{bond_slaves}")
+              network_options.push("--bondopts=mode=#{iface.mode},#{iface.bond_options.tr(' ', ',')}")
+            end
+
+            # VLAN (only on physical is recognized)
+            if iface.virtual? && iface.tag.present? && iface.attached_to.present?
+              if rhel_compatible
```

Comment:
https://pykickstart.readthedocs.io/en/latest/kickstart-docs.html#network states Fedora 19 introduced this, so  please drop this condition:
```suggestion
```

It may be that EL6 doesn't support it, but this current line doesn't say anything about that.